### PR TITLE
lauren fries brink : set up project baseline

### PR DESCRIPTION
**Pull request by laurenfb**
i think this does it!

### CODE REVIEW

**Comment by laurenfb** [comment]
@PilgrimMemoirs @CheezItMan  please approve me :)

**Comment by CheezItMan** [comment]
@laureneliz Looking at it now

File: farmar.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'csv'
+
+module FarMar
```
Comment (CheezItMan) [inline farmar.rb:3]: Should this end before the requires_relatives?

**Comment by CheezItMan** [comment]
@laureneliz It looks ok, just see my note on the the far_mar.rb file, the require_relative commands are inside the module.

**Pull request closed without merging**